lib/bolt/cli.rb

```diff
@@ -183,6 +187,10 @@ def validate(options)
         raise Bolt::CLIError,
               "Option '--noop' may only be specified when running a task"
       end
+
```

Comment:
We should also add a check that `options[:object]` is `-e` or `--execute` - I don't think this will be common, but puppet users might try other puppet apply flags with this command.

Reply:
`options[:code]` is set to the _value_ of the `-e/--execute` flag in the hook in `BoltOptionParser`. It will only be set if they used that flag.